src/client/java/teammates/client/scripts/testdataconversion/ConvertDatastoreJsonToSqlJson.java

```diff
@@ -0,0 +1,215 @@
+            String jsonKey = removeWhitespace(student.getSection());
+
+            if (!sqlDataBundle.sections.containsKey(jsonKey)) {
+                Section sqlSection = entityConverter.createSection(student);
+                sqlDataBundle.sections.put(jsonKey, sqlSection);
+            }
+        });
+
+        dataStoreBundle.students.forEach((k, student) -> {
+            // Assumes that team name is unique in JSON file
+            String jsonKey = removeWhitespace(student.getTeam());
+
+            if (!sqlDataBundle.teams.containsKey(jsonKey)) {
+                Team sqlTeam = entityConverter.createTeam(student);
+                sqlDataBundle.teams.put(k, sqlTeam);
+            }
+        });
+
+        dataStoreBundle.instructors.forEach((k, instructor) -> {
+            Instructor sqlInstructor = entityConverter.convert(instructor);
+            sqlDataBundle.instructors.put(k, sqlInstructor);
+        });
+
+        dataStoreBundle.students.forEach((k, student) -> {
+            Student sqlStudent = entityConverter.convert(student);
+            sqlDataBundle.students.put(k, sqlStudent);
+        });
+
+        dataStoreBundle.deadlineExtensions.forEach((k, deadlineExtension) -> {
+            DeadlineExtension sqlDeadline = entityConverter.convert(deadlineExtension);
+            sqlDataBundle.deadlineExtensions.put(k, sqlDeadline);
+        });
+
+        dataStoreBundle.feedbackQuestions.forEach((k, feedbackQuestion) -> {
+            FeedbackQuestion sqlFeedbackQuestion = entityConverter.convert(feedbackQuestion);
+            sqlDataBundle.feedbackQuestions.put(k, sqlFeedbackQuestion);
+        });
+
+        dataStoreBundle.accounts.forEach((k, account) -> {
+            List<ReadNotification> sqlReadNotifications = entityConverter.createReadNotifications(account);
+            sqlReadNotifications.forEach(notif -> {
+                String jsonKey = removeWhitespace(notif.getNotification().getTitle() + "-" + account.getName());
+                sqlDataBundle.readNotifications.put(jsonKey, notif);
+            });
+        });
+
+        dataStoreBundle.feedbackResponses.forEach((k, feedbackResponse) -> {
+            FeedbackResponse sqlFeedbackResponse = entityConverter.convert(feedbackResponse);
+            sqlDataBundle.feedbackResponses.put(k, sqlFeedbackResponse);
+        });
+
+        dataStoreBundle.feedbackResponseComments.forEach((k, feedbackReponseComment) -> {
+            FeedbackResponseComment sqlFeedbackResponseComment = entityConverter.convert(feedbackReponseComment);
+            sqlDataBundle.feedbackResponseComments.put(k, sqlFeedbackResponseComment);
+        });
+    }
+
+    public static void main(String[] args) throws IOException, InvalidParametersException {
+        if (args.length > 0) {
+            File inputFile = new File(args[0]);
+            ConvertDatastoreJsonToSqlJson script = new ConvertDatastoreJsonToSqlJson(inputFile);
+            String outputFileName = "output.json";
```

Comment:
could we have the outputFileName be <name of .json>SQLTest.json, (e.g. For InstructorHomePageE2ETest.json, have it output: InstructorHomePageE2ESqlTest.json or InstructorHomePageE2ETestSql.json (this is easier to do))